src/SharpMetal.Generator/Namespaces.cs

```diff
@@ -2,7 +2,7 @@ namespace SharpMetal.Generator
 {
     public static class Namespaces
     {
-        public static string[] Prefixes { get; } = ["MTLFX", "MTL", "NS", "CA", "CF", "CG", "IO"];
+        public static string[] Prefixes { get; } = ["MTLFX", "MTL4FX", "MTL", "MTL4", "NS", "CA", "CF", "CG", "IO", "kern" ];
```

Comment:
I think the space after `"kern"` is causing the format failure.

Reply:
Fixed the formatting, sorry about that :)